src/ert/job_queue/queue.py

```diff
@@ -102,7 +100,18 @@ def __repr__(self) -> str:
     def __str__(self) -> str:
         return self.__repr__()
 
-    def __init__(self, queue_config: QueueConfig):
+    def __init__(
+        self,
+        queue_config: QueueConfig,
+        realizations: Optional[Sequence[Realization]] = None,
+        *,
```

Comment:
Should not this star be one line up, as `realizations` looks like a named argument to me?

Reply:
If it forces realizations to be specified before ens_id, it is perfectly fine with me.